### ISSUE TITLE
Documentation issue: C / C   Rules -- I cannot parse meaning from `include_prefix`

### ISSUE THREAD

**Issue by tgockel** [bazelbuild/bazel#12043]
Documentation URL: https://docs.bazel.build/versions/master/be/c-cpp.html

From the documentation for `include_prefix`:

> When set, the headers in the `hdrs` attribute of this rule are accessible at is the value of this attribute prepended to their repository-relative path. 

I have read this sentence over and over again, but it mystifies me -- my brain gives up at "at is the value." I would offer a way to rephrase it to be less perplexing, but unfortunately I don't understand how `include_prefix` works because I can't understand the documentation for it. I'm stuck in a bit of a loop on this one.

**Comment by benjaminp** [comment]
The "is" seems erroneous.

**Comment by tgockel** [comment]
> When set, the headers in the `hdrs` attribute of this rule are accessible at the value of this attribute prepended to their repository-relative path.

Is this saying that:

```
. proj
|- foo.hpp
|- bar.cpp
|- BUILD
|- WORKSPACE
```

With `BUILD`:

```
cc_library(
    hdrs = ['foo.hpp'],
    srcs = ['bar.cpp'],
    include_prefix = 'blop',
)
```

Then `bar.cpp`:

```
#include <blop/foo.hpp>

...
```

?
